src/UserManager.ts

```diff
@@ -329,12 +329,14 @@ export class UserManager {
             logger.throw(new Error("No silent_redirect_uri configured"));
         }
 
+        const user: User = await this.getUser() as User;
```

Comment:
const user = await this.getUser();

getUser returns User or null in promise, no need to cast with as....